.github/workflows/polaris-react-ci.yml

```
@@ -0,0 +1,94 @@
+name: '@shopify/polaris CI'
+
+on:
+  push:
+    branches: [main, polaris-release]
+    paths:
+      - 'polaris-react/**'
+  pull_request:
+    paths:
+      - 'polaris-react/**'
```

```suggestion
      - 'polaris-react/src/**'
      - 'polaris-tokens/src/**'
  pull_request:
    paths:
      - 'polaris-react/src/**'
      - 'polaris-tokens/src/**'
```

@bpscott made a good point that tokens would also benefit from running these actions.

---

@samrose3 If we want this maybe it should be called `a11y-vrt-ci.yml`